### ISSUE TITLE
[APP] Helium Console

### ISSUE THREAD

**Issue by davetapley** [PipedreamHQ/pipedream#1165]
**Name of app / service**
[Helium Console](https://www.helium.com/console)

**Link to developer documentation**
https://docs.helium.com/use-the-network/console/integrations/

**Is lack of support preventing you from building workflows, or do you have a workaround?**

No, HTTP web hook works okay for now.

**Are there specific actions, or triggers, you'd like to see for this app? Please let us know here or use the Action and Trigger issue templates to open requests for each!**

Helium console allows for _uplink_ (from device to network) and _downlink_ (from network to device), these are analogous to trigger and action respectively. Uplink (trigger) would probably cover 99% of use cases.

**Comment by todsac** [comment]
Just to clarify, would this integration be for the Console API (as opposed to the Blockchain API)?  Or, is it something different entirely?

**Comment by todsac** [comment]
Here is the Helium console API integrated - https://pipedream.com/apps/helium

**Comment by davetapley** [comment]
@todsac console.

Whoa, did you _just_ do that, or did I miss it? 🤪

**Comment by davetapley** [comment]
Oh, wait, so that's a start, but probably the most useful case would be specifically this:
https://docs.helium.com/use-the-network/console/integrations/http/

It's something you can set up manually using a Pipedream HTTP endpoint as a trigger, but it would be neat if Pipedream was an 'officially supported' integration on the Helium Console side (i.e. appearing in this ⬇️ list).

![](https://docs.helium.com/img/use-the-network/console/integrations/integrations-select-http.png)

**Comment by todsac** [comment]
Yes, just did it.  I can look into the native integration options but won't be as fast :-)

**Comment by davetapley** [comment]
Thanks @todsac, TIL the Console is open source too, so I opened an issue over there ⬆️

**Comment by davetapley** [comment]
@todsac got the 👍🏻 on the other side: https://github.com/helium/console/issues/514#issuecomment-822895609

I won't have time to get on it for next few weeks, but could be good visibility for Pipedream if that's an incentive 😁

**Comment by jdgemm** [comment]
@davetapley @todsac any update?
